Examples/Algorithms/Vertexing/src/IterativeVertexFinderAlgorithm.cpp

```diff
@@ -90,26 +90,24 @@ ActsExamples::ProcessCode ActsExamples::IterativeVertexFinderAlgorithm::execute(
   // Set up propagator with void navigator
   auto propagator = std::make_shared<Propagator>(
       stepper, Acts::detail::VoidNavigator{}, logger().cloneWithSuffix("Prop"));
-  PropagatorOptions propagatorOpts(ctx.geoContext, ctx.magFieldContext);
   // Setup the vertex fitter
-  VertexFitter::Config vertexFitterCfg;
-  VertexFitter vertexFitter(vertexFitterCfg);
+  Fitter::Config vertexFitterCfg;
+  Fitter vertexFitter(vertexFitterCfg);
   // Setup the track linearizer
   Linearizer::Config linearizerCfg(m_cfg.bField, propagator);
   Linearizer linearizer(linearizerCfg, logger().cloneWithSuffix("HelLin"));
   // Setup the seed finder
-  ImpactPointEstimator::Config ipEstCfg(m_cfg.bField, propagator);
-  ImpactPointEstimator ipEst(ipEstCfg);
-  VertexSeeder::Config seederCfg(ipEst);
-  VertexSeeder seeder(seederCfg);
+  IPEstimator::Config ipEstCfg(m_cfg.bField, propagator);
+  IPEstimator ipEst(ipEstCfg);
+  Seeder seeder;
```

Comment:
No config anymore?

Reply:
I aligned this with the AMVF. Now we use the `TrackDensityVertexFinder` which seems to go fine with the default constructed `GaussianTrackDensity`